pkg/cli/cluster_free_disk_space.go

```diff
@@ -0,0 +1,133 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
+)
+
+const (
+	// defaultOpenEBSPodImage is the image used during disk free check for openebs storage.
+	// this is the same image used by the pvmigrate project, it may be any image containing
+	// 'df' and 'cat' commands.
+	defaultOpenEBSPodImage          = "eeacms/rsync:2.3"
+	isDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
+	openEBSLocalProvisioner         = "openebs.io/local"
+	rookRBDProvisioner              = "rook-ceph.rbd.csi.ceph.com"
+	rookCephFSProvisioner           = "rook-ceph.cephfs.csi.ceph.com"
+)
+
+// NewClusterCheckFreeDiskSpaceCmd returns a command that is capable of reporting back the amount of free space in the cluster
+// for a provided storage class.
+func NewClusterCheckFreeDiskSpaceCmd(cli CLI) *cobra.Command {
+	var forStorageClass, openEBSImage, openEBSNode string
+	logbuf := bytes.NewBuffer(nil)
+	logger := log.New(logbuf, "", 0)
+	k8sConfig := config.GetConfigOrDie()
+	clientSet := kubernetes.NewForConfigOrDie(k8sConfig)
+	rookClientSet := rookcli.NewForConfigOrDie(k8sConfig)
+
+	cmd := &cobra.Command{
+		Use:          "check-free-disk-space",
+		Short:        "Returns the amount of free disk space (in bytes) for a given storage class",
+		Example:      "kurl cluster check-free-disk-space --storageclass openebs",
```

Comment:
```suggestion
		Example:      "kurl cluster check-free-disk-space --storageclass openebs",
```

Can we improve this example by adding all possible flags with values? So that we know how to use it?

Reply:
Done.